server/datastore/mysql/apple_mdm.go

```diff
@@ -3513,3 +3513,21 @@ WHERE
 
 	return &res, nil
 }
+
+func (ds *Datastore) InsertMDMAppleDDMRequest(ctx context.Context, hostUUID, message_type, raw_json string) error {
```

Comment:
Nit, but for more idiomatic Go naming those should be `messageType` and `rawJSON`. (for reference: https://go.dev/doc/effective_go#names)

Reply:
Fixed, I had just written the schema so I still had snake_case muscle memory